Examples/Volume/VolumeMapperBlendModes/index.js

```diff
@@ -31,10 +31,12 @@ fullScreenRenderer.addController(controlPanel);
 // ----------------------------------------------------------------------------
 // Example code
 // ----------------------------------------------------------------------------
-// Server is not sending the .gz and with the compress header
-// Need to fetch the true file name and uncompress it locally
-// ----------------------------------------------------------------------------
-
+let currentBlendMode = null;
```

Comment:
why bother exposing this variable when you can replace it with `mapper.getBlendMode()`